docs/titan-basics/workspace-windows.md

```diff
@@ -275,7 +275,7 @@ button at a glance. Select the <Keys.SoftKey>Halo</Keys.SoftKey> option on the S
 
 If you set a [halo on fixture buttons](../patching/changing-the-patch.md#fixture-button-halo), the halo colour is also used in other windows; fixture buttons can also have an automatic halo to help identify different types of fixture, which is also used in the patch window.
 
-On the Diamond 9, the halo colour set on playbacks is used in the level bargraph lighting and the playback indicator.
+On the Diamond 9 and Diamond 7, the halo colour set on playbacks is used in the level bargraph lighting and the playback indicator.
```

Comment:
I think bar graph is normally 2 words?

As previously mentioned that isn't what is on the D7 but this sentence could be interpreted either way.